arelle/plugin/validate/NL/rules/nl_kvk.py

```diff
@@ -818,3 +820,30 @@ def rule_nl_kvk_3_6_3_3(
                   'Allowed characters include: A-Z, a-z, 0-9, underscore ( _ ), period ( . ), and hyphen ( - ). '
                   'Update filing naming to review unallowed characters. '
                   'Invalid filenames: %(invalidBasenames)s'))
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NL_INLINE_2024
+    ],
+)
+def rule_nl_kvk_6_1_1_1(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    NL-KVK.6.1.1.1: The size of the report package MUST NOT exceed 100 MB.
+    """
+    if val.modelXbrl.fileSource.fs: # must be a zip to be a report package
+        assert isinstance(val.modelXbrl.fileSource.fs, zipfile.ZipFile)
```

Comment:
`assert` seems like it will fail badly if it's not a zip, so it seems like the wrong thing to do.  Perhaps `Validation.warning` that the report package size cannot be determined?